tron/core/job.py

```diff
@@ -176,12 +180,22 @@ def state_data(self):
         """This data is used to serialize the state of this job."""
         return {
             'runs': self.runs.state_data,
-            'enabled': self.enabled,
+            'enabled': {
+                'current': self.enabled,
+                'config': self.config_enabled,
+            }
         }
 
     def get_job_runs_from_state(self, state_data):
         """Apply a previous state to this Job."""
-        self.enabled = state_data['enabled']
+        if self.config_enabled is not state_data['enabled']['config']:
```

Comment:
is difference between `False` and `None` important here?

Reply:
It would've been, but given @qui's comment about initializing config_enabled, it can never be None now (unless self.enabled is somehow set to None).

Reply:
Yes, but neither should ever be `None` here, at least not now. Based on @qui's comments above, I've changed it so that `self.config_enabled` = `self.enabled` during initialization.